src/decorators.js

```diff
@@ -55,10 +55,15 @@ export function customElement(name: string): any {
 /**
 * Decorator: Indicates that the decorated class is a custom attribute.
 * @param name The name of the custom attribute.
-* @param defaultBindingMode The default binding mode to use when the attribute is bound wtih .bind.
+* @param defaultBindingMode The default binding mode to use when the attribute is bound with .bind.
+* @param aliases The array of aliases to associate to the custom attribute.
 */
-export function customAttribute(name: string, defaultBindingMode?: number): any {
+export function customAttribute(name: string, defaultBindingMode?: number, aliases?: string[]): any {
   return function(target) {
+    if (aliases) {
+      target.aliases = aliases;
```

Comment:
the reason is the use case for i18n, where the developer needs a chance to modify the aliases from outside. As such its easy to import the TCustomAttribute and overload the static property. It would be quite cumbersome to get access to the `metadata`, being honest right now I wouldn't even know how to do that.